### PR TITLE
Add job, executor and example for openapi validation

### PR DESCRIPTION
**Pull request by dtvalk-ov**
Add functionality to validate swagger files using the swagger-cli. The path to the file that needs validation is configurable and it has a default value './api/api.yml'
https://financialtimes.atlassian.net/browse/UPPSF-2500

### CODE REVIEW

File: src/examples/use-openapi-validate.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: Example using orb's job openapi-validate with default node executor passing custom file name parameter in order to validate an openapi swagger file
+usage:
+  version: 2.1
+  orbs:
+    golang-ci: epavlova/golang-ci@0.2.0
```
Comment (karsov) [inline src/examples/use-openapi-validate.yml:5]: Should be
```suggestion
    golang-ci: financial-times/golang-ci@1
```

**Review by karsov — COMMENTED** [review]
I think we should fix the node & swagger-cli versions so that we don't have it unexpectedly stop working one day.

**Comment by dtvalk-ov** [comment]
> I think we should fix the node & swagger-cli versions so that we don't have it unexpectedly stop working one day.

Added fixed versions

**Comment by AlexKondov** [comment]
https://github.com/Financial-Times/upp-image-validator/pull/7 - here's a PR that tests the orb in the image validator

**Comment by dtvalk-ov** [comment]
@gotha Tested w/ branches https://github.com/Financial-Times/upp-image-validator/tree/test/orb-opebapi-test & https://github.com/Financial-Times/upp-image-validator/tree/test/orb

**Review by karsov — APPROVED** [review]

**Review by gotha — APPROVED** [review]

**Pull request merged**